test/FunctionalTests/project.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": true
+  },
+  "commands": {
+    "test": "xunit.runner.aspnet"
+  },
+  "dependencies": {
+    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-*",
```

Comment:
I don't think you need IISPlatformHandler in the test launcher, just in the samples.